### DISCUSSION TITLE
The famous saying, "The lady doth protest too much, methinks" is from which play by Shakespeare?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#113] [Galaxy Trivia] [answered]
**Hint:** 💀👈

**Comment by Tynab** [comment, marked as answer]
The famous saying, "The lady doth protest too much, methinks" is from the play "Hamlet" by William Shakespeare. The line is spoken by Queen Gertrude during the play within the play, which is used to expose the guilt of King Claudius in the murder of Hamlet's father.
